service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/LegalRepresentativeAddedContentProviderTest.java

```diff
@@ -52,10 +52,14 @@
     @Mock
     private CaseUrlService caseUrlService;
 
-    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration = mock(LocalAuthorityNameLookupConfiguration.class);
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration = mock(
```

Comment:
could this not be annotated with `@Mock`

Reply:
I had to inject it directly since some funkiness with the injection of the abstract class :(